src/test/BondingManagerFuzzer.sol

```diff
@@ -0,0 +1,95 @@
+
+    function startPrank(address) external;
+
+    function prank(address) external;
+
+    function stopPrank() external;
+
+    function assume(bool) external;
+}
+
+// forge test -vvv --fork-url <ARB_MAINNET_RPC_URL> --fork-block-number 6768456 --match-contract BondingManagerFuzzer
+contract BondingManagerFuzzer is DSTest {
+    CheatCodes public constant CHEATS = CheatCodes(HEVM_ADDRESS);
+
+    BondingManager public constant BONDING_MANAGER = BondingManager(0x35Bcf3c30594191d53231E4FF333E8A770453e40);
+    address public constant MINTER_ADDRESS = 0xc20DE37170B45774e6CD3d2304017fc962f27252;
+    LivepeerToken public constant LIVEPEER_TOKEN = LivepeerToken(0x289ba1701C2F088cf0faf8B3705246331cB8A839);
+    RoundsManager public constant ROUNDS_MANAGER = RoundsManager(0xdd6f56DcC28D3F5f27084381fE8Df634985cc39f);
+    Controller public constant CONTROLLER = Controller(0xD8E8328501E9645d16Cf49539efC04f734606ee4);
+
+    address public constant delegator = 0xF8E893C7D84E366f7Bc6bc1cdB568Ff8c91bCF57;
+    address public constant delegatorB = 0x5bE44e23041E93CDF9bCd5A0968524e104e38ae1;
+    address public constant delegate = 0xDcd2CD1a27118E65A3d8aF6518F62b78D056Ac5a;
+    uint256 public constant STARTING_BLOCK = 14265594;
+
+    function testTransferBond(uint256 _amount) public {
+        CHEATS.assume(_amount > 0.01 ether && _amount < 100000 ether);
+        CHEATS.roll(STARTING_BLOCK);
+        CHEATS.prank(delegatorB);
+        BONDING_MANAGER.claimEarnings(0);
+        CHEATS.prank(delegator);
+        BONDING_MANAGER.claimEarnings(0);
+
+        CHEATS.prank(MINTER_ADDRESS);
+        LIVEPEER_TOKEN.mint(delegator, _amount);
+
+        CHEATS.startPrank(delegator);
+        (uint256 initialBondedAmount, , address initialDelegateAddress, , , , ) = BONDING_MANAGER.getDelegator(
+            delegator
+        );
+        LIVEPEER_TOKEN.approve(address(BONDING_MANAGER), _amount);
+        BONDING_MANAGER.bond(_amount, delegate);
+
+        // time-travels to new new round and initializes it
+        CHEATS.roll(STARTING_BLOCK + ROUNDS_MANAGER.roundLength());
+        ROUNDS_MANAGER.initializeRound();
+        (uint256 updatedBondedAmount, , address updatedDelegateAddress, , , , ) = BONDING_MANAGER.getDelegator(
+            delegator
+        );
+        assertEq(updatedBondedAmount, initialBondedAmount + _amount);
+        assertEq(updatedDelegateAddress, delegate);
+
+        (uint256 initialRecipientBondedAmount, , address initialRecipientDelegateAddress, , , , ) = BONDING_MANAGER
+            .getDelegator(delegatorB);
+
+        BondingManager(BONDING_MANAGER).transferBond(
+            delegatorB,
+            _amount,
+            address(0),
+            address(0),
+            address(0),
+            address(0)
+        );
+
+        // // time-travels to new new round and initializes it
+        CHEATS.roll(block.timestamp + ROUNDS_MANAGER.roundLength() * 2);
+        ROUNDS_MANAGER.initializeRound();
```

Comment:
Delegator's bonded amounts will be reflected in the same round, so we can skip this part. Doesn't hurt to keep it as is though but it saves a bit of runtime.